CONTRIBUTING.md

```diff
@@ -1,19 +1,43 @@
 # Contributing to Charts
 
-Yippee! If you're here, you _must_ be interested in contributing something to Charts—and if that's the case, it's safe to say that one: we like you already 🤓 and two: we would like to personally thank you ahead of time ❤️
+Yippee! If you're here, you _must_ be interested in contributing something to Charts.
+
+If that's the case, we would like to personally thank you ahead of time 🤓❤️
 
 Please take a moment to review this document in order to make the contribution process easy and effective for everyone involved.
 
 ## Table of contents
 
+- **[Run Charts in Local Environment](#run-charts-in-local-environment)**
 - [Issue guidelines](#issue-guidelines)
 - [Pull request guidelines](#pull-request-guidelines)
   - [Contribution process](#contribution-process)
-  - [NPM commands](#npm-commands)
 - [Code guidelines](#code-guidelines)
-  - [Resources](#resources)
   - [Philosophy](#philosophy)
 
+
+## Run Charts in Local Environment
+We use Lerna to manage all packages in Charts.
+
+**These are the Node & NPM versions we recommend:**
+- Node Version: `8.11.4`
+- NPM Version: `6.4.1`
+
+To get charts running locally on your machine:
+1. Fork this repo
+2. Clone your fork
+3. Run `npm install` (this would install all packages and get **lerna** setup)
+
+The **core** package (vanilla JS) demos can be launched using:
+```sh
```

Comment:
Add line before for readability in code if it doesn't make a change in rendering.
```suggestion

```sh
```